### PR TITLE
[Scala]: Cask framework added

### PR DESCRIPTION
**Pull request by whiplash**

### CODE REVIEW

**Comment by waghanza** [comment]
cc @lihaoyi

**Comment by whiplash** [comment]
Hmmm... I think I rushed out too much. Let me see what goes wrong.

File: scala/cask/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```
Comment (waghanza) [inline scala/cask/project/build.properties:1]: maybe this file could dropped.

the sbt version is driven by dockerfile, actually it is `1.10.2`

**Review by waghanza — COMMENTED** [review]
Can you finish it for today @whiplash ? to have it in results tomorrow

**Comment by lihaoyi** [comment]
go for it

**Comment by whiplash** [comment]
I can't find out what the problem is. @waghanza can you remind me again how one runs it locally?

**Comment by waghanza** [comment]
@whiplash On linux, I run
```
export FWK=scala/cast && export LANGUAGE=`echo $FWK | awk -F '/' '{print $1}'` && export FRAMEWORK=`echo $FWK | awk -F '/' '{print $2}'` && cd ~/web-frameworks && yes|docker system prune -a && rm -fr Gemfile.lock && bundle install && bundle exec rake config && make build -f $FWK/.Makefile && sleep 60 && make collect -f $FWK/.Makefile && bundle exec rake db:raw_export && bundle exec rake db:check_failures
```

but maybe you can log onto the container and run the server

**Comment by waghanza** [comment]
seems just a false positove of CI, I'll merge and include it

**Comment by waghanza** [comment]
no, it was not that. maybe a library issue. perhaps, by adding logs we can debug @whiplash

**Comment by waghanza** [comment]
Was included @whiplash 
https://web-frameworks-benchmark.netlify.app/result?l=scala

**Pull request merged**